spec/form_models/offender_sar_case_form_spec.rb

```diff
@@ -80,8 +80,8 @@
 
     context "when params is set and valid" do
       let(:params) { ActionController::Parameters.new(
-        case_form: { subject_type: "offender", date_of_birth: "2001-01-01", flag_for_disclosure_specialists: "no" }
-      ).require(:case_form).permit(:subject_type, :date_of_birth, :flag_for_disclosure_specialists)}
+        case_form: { subject_type: "offender", date_of_birth: "2001-01-01", flag_as_high_profile: "no" }
+      ).require(:case_form).permit(:subject_type, :date_of_birth, :flag_as_high_profile)}
 
```

Comment:
This is good, is there a 'negative' test for gibberish or nil `flag_as_high_profile` value?

Reply:
Thanks, Mo, I've beefed up the model spec and added more coverage for these